test/extended/registry/util/util.go

```diff
@@ -120,13 +120,16 @@ func GetRegistryPod(podsGetter kcoreclient.PodsGetter) (*kapiv1.Pod, error) {
 	return &podList.Items[0], nil
 }
 
-// LogRegistryPod attempts to write registry log to a file to recent test's output directory.
+// LogRegistryPod attempts to write registry log to a file in artifacts directory.
 func LogRegistryPod(oc *exutil.CLI) error {
 	pod, err := GetRegistryPod(oc.KubeClient().Core())
 	if err != nil {
 		return fmt.Errorf("failed to get registry pod: %v", err)
 	}
-	path, err := oc.Run("logs").Args("dc/docker-registry").OutputToFile("pod-" + pod.Name + ".log")
+
+	ocLocal := *oc
+	ocLocal.SetOutputDir(exutil.ArtifactDirPath())
```

Comment:
is the artifacts dir gathered?  it's not obvious to me that it is: https://ci.openshift.redhat.com/jenkins/me/my-views/view/DevEx/job/test_branch_origin_extended_builds/238/s3/

Reply:
I believe `exutil.ArtifactDirPath()` returns a root dir for the artifacts - the one you can see in this view: https://ci.openshift.redhat.com/jenkins/view/Origin%20Test%20Jobs/job/test_branch_origin_extended_builds/238/s3/

I can see the artifacts dir in the view of prow runs: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/16859/test_pull_request_origin_extended_image_registry/36/

Reply:
ok, i was thinking it was the testadata fixture path, i guess i'm wrong.